src/Catalog/Persistence/AzureCloudBlockBlob.cs

```diff
@@ -13,35 +13,30 @@ namespace NuGet.Services.Metadata.Catalog.Persistence
 {
     public sealed class AzureCloudBlockBlob : ICloudBlockBlob
     {
-        private readonly CloudBlockBlob _blob;
+        public CloudBlockBlob Blob { get; }
 
```

Comment:
Super nit: usually properties are placed after the constructors. Same for `ETag` and `Uri`.